src/scrcmd.c

```diff
@@ -5443,12 +5443,12 @@ static BOOL ScrCmd_187(ScriptContext *ctx)
     return FALSE;
 }
 
-static BOOL ScrCmd_188(ScriptContext *ctx)
+static BOOL ScrCmd_SetObjectEventMovType(ScriptContext *ctx)
```

Comment:
```suggestion
static BOOL ScrCmd_SetObjectEventMovementType(ScriptContext *ctx)
```

Reply:
Should this also apply to the script macro too?

Reply:
Yes obviously, it makes no sense for them to have different names.

Reply:
> Yes obviously, it makes no sense for them to have different names.

done 👍